### PR TITLE
Fix data type issue in searching/training configurations

### PR DESCRIPTION
**Pull request by dongyang0122**
Signed-off-by: dongyang0122 <don.yang.mech@gmail.com>

### Description
Fix data type issue in searching/training configurations of DiNTS.

### Status
Ready

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json`.
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by wyli — APPROVED** [review]

File: models/pancreas_ct_dints_segmentation/configs/search.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ transform_train:
     - label4crop
   - _target_: Lambdad
     keys: label4crop
-    func: "$lambda x, s=@output_classes: np.concatenate(tuple([ndimage.binary_dilation((x==_k).astype(x.dtype), iterations=48).astype(x.dtype) for _k in range(s)]), axis=0)"
+    func: "$lambda x, s=@output_classes: np.concatenate(tuple([ndimage.binary_dilation((x==_k).astype(x.dtype), iterations=48).astype(float) for _k in range(s)]), axis=0)"
```
Comment (wyli) [inline models/pancreas_ct_dints_segmentation/configs/search.yaml:88]: Please also update the metadata, change monai dep to 0.9.1, and the package version to 0.1.1

File: models/pancreas_ct_dints_segmentation/configs/metadata.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.1.0",
+    "version": "0.1.1",
     "changelog": {
         "0.1.0": "complete the model package",
```
Comment (Nic-Ma) [inline models/pancreas_ct_dints_segmentation/configs/metadata.json:6]: Please also add the changelog of 0.1.1.

Thanks.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**